### ISSUE TITLE
[BUG] Steps skipping or sudden current variation while traveling

### ISSUE THREAD

**Issue by Andre1902** [MarlinFirmware/Marlin#24979]
### Did you test the latest `bugfix-2.1.x` code?

Yes, and the problem still exists.

### Bug Description

i can see on long travels some random steps skipping (X and Y). in the last few weeks i tried with changing every possible setting  i know in the firmware but i cant figure out the problem (such as acceleration, speed, current, sensorless homing, silent stepper driver, interpolation...). i sliced with other slicer and with cura i could manage to get some better results, maybe due to the acceleration and jerk control, but with prusa slicer i couldnt find a solution to the issue. i also tried to change some marlin version from 2.x up to the last bugfix.

### Expected behavior

after motherboard ubgrate from Creality 4.2.2 to BTT SKR Mini E3 V3.0

### Version of Marlin Firmware

Marlin-bugfix-2.1.x

### Printer model

Ender 3

### Electronics

BTT SKR Mini E3 V3.0

### Bed Leveling

ABL Bilinear mesh

### Your Slicer

Prusa Slicer

### Don't forget to include

- [X] A ZIP file containing your `Configuration.h` and `Configuration_adv.h`.

### Additional information & file uploads

[config.zip](https://github.com/MarlinFirmware/Marlin/files/9995475/config.zip)

**Comment by oliver-eifler** [comment]
You should deactivate S_CURVE_ACCELERATION - see https://github.com/MarlinFirmware/Marlin/pull/24955 for explanation

and set the currents according to your motors

**Comment by Andre1902** [comment]
Ive also tryed to disable it, current was set to 800 according to my TMC2209, a little bit higher than marlin default for ender 3. I solved my issue yesterday by reading this #19217. Deactivated SEGMENT_LEVELED_MOVES and now it works fine, no more stuttering.

**Comment by Fredrossi046** [comment]
Problem is not possible to be solved for me, because i have the pre-compiled firmware. 
I installed this firmware : firmware_Ender6_BLT_50W_ET_DW7.4.6.bin.zip  from  insanityautomation.com/index.php/firmware-projects

Machine was working fine with small prints, but now i get the stuttering problem as well, at least with larger prints. Disable ABL is all i can do, but that won't solve it.  Deactivating SEGMENT_LEVELED_MOVES is not possible for me, as i can not open the bin-file in Arduino-IDE. Or is there a way i can open, alter and place it back on SD for updating firmware?

I changed current for the stepper driver (at mainboard), but that is all i can do i guess. Strange thing is, i printed a feedrate test, that is larger than the bed level points of my 25 ABL points, but that did not have any problems. 

Mainboard is stock, just the above update with initial no problems last few days.

Edit: if anyone could provide the above code, i would be able to acces the mainboard and change it in IDE. I would prefer to have it that way, so i could make changes provided here instead of waiting for the complete package.
